web/src/components/organisms/Settings/workspaceList/index.tsx

```diff
@@ -0,0 +1,93 @@
+import styled from "@emotion/styled";
+import Button from "@reearth-cms/components/atoms/Button";
+import Form from "@reearth-cms/components/atoms/Form";
+import Input from "@reearth-cms/components/atoms/Input";
+import List from "@reearth-cms/components/atoms/List";
+import { Content } from "antd/lib/layout/layout";
+import { useCallback, useState } from "react";
+import { useNavigate } from "react-router-dom";
+
+import useHooks from "../Workspace/hooks";
+
+type Props = {
+  workspaceId: string;
+};
+
+const WorkspaceList: React.FC<Props> = ({ workspaceId }) => {
+  const { workspaces, createWorkspace, selectWorkspace } = useHooks({
+    workspaceId,
+  });
+  const navigate = useNavigate();
+  const [workspaceName, setWorkspaceName] = useState<string>("");
```

Comment:
For useState that are strings, you do both `useState("")` and `useState<string>("")`. Can we stick to one (preferably the first one, since the type can be inferred by the "")??